loadclient/config.go

```diff
@@ -0,0 +1,15 @@
+package loadclient
+
+type Options struct {
+	Threads              int
+	LogLinesPerSec       int
+	Destination          string
+	Source               string
+	SyntheticPayloadSize int
+	TotalLogLines        int
+	LogFormat            string
+	OutputFile           string
+	DestinationAPIURL    string
+	LokiTenantID         string
+	LokiLabels           string
```

Comment:
It would nice to extract Loki specific configuration in a separate struct and make it a point here, e.g.:
```go
type Options struct {
  LokiConfig *Loki
}

type Loki struct {
  TenantID string
  Labels map[string]string
}
```

Reply:
Fixed
DONE